### PR TITLE
XRT-1696: Tidy up azure configs & docs

### PR DESCRIPTION
**Pull request by amaan-akram**
will update modbus example and docs under https://iotechsys.atlassian.net/jira/software/projects/XRT/boards/65?assignee=60a22dc797f3d4007011cf77&selectedIssue=XRT-1679

### CODE REVIEW

File: AzureSphere/docs/ethernetip-example.md [hunk 1]
```diff
@@ -1,111 +1,156 @@
...
+the device twin are automatically downloaded and persisted to the
 Azure Sphere device. An example set of "desired" properties can be
-found at:
-[twin/desired-ethernetip.json](../twin/desired-ethernetip.json)
+found at: [twin/desired-ethernetip.json](../twin/desired-ethernetip.json)
 
 The Microsoft Azure Device Twin corresponding to the Azure Sphere
 development board will need updating to include the same "azure"
 component settings. This is found in the Device twin JSON at
 "properties" / "desired" / "Components" / "azure". This is can be
```
Comment (BinduRao2018) [inline AzureSphere/docs/ethernetip-example.md:107]: typo: This is can be

File: AzureSphere/docs/ubuntu-installation.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Just like with the [windows installation](./windows-installation.md) we can manu
 
 2. Download XRT using:
 
-```bat
+```bash
 curl -L -O https://iotech.jfrog.io/ui/native/windows-release/azuresphere-12/iotech-xrt-dev-azsphere12-2.0-latest.zip
```
Comment (BinduRao2018) [inline AzureSphere/docs/ubuntu-installation.md:48]: Why do we suffix the image with 'latest'?

File: AzureSphere/docs/windows-installation.md [hunk 3]
```diff
@@ -33,5 +33,4 @@ or via the Visual Studio Command Prompt.
 
 ## Updating the XRT package on Windows
 
-When updating the xrt packages with a newer version, it is recomended that you reinstall the azure sphere SDK to ensure we have a clean sysroots directory. After this the same steps can be followed for [installing](#installing-the-xrt-package-on-windows). 
-
+When updating the xrt packages with a newer version, it is recommended that you reinstall the azure sphere SDK to ensure we have a clean sysroots directory. After this the same steps can be followed for [installing](#installing-the-xrt-package-on-windows).
```
Comment (BinduRao2018) [inline AzureSphere/docs/windows-installation.md:36]: Not clear. Are we saying about the update or fresh install?

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]
Have left few inline comments

File: AzureSphere/docs/bacnet-example.md [hunk 4]
```diff
@@ -25,14 +25,35 @@ Note: The prerequisites found on the main [readme.md](../README.md) are also req
...
+The configurations that are required to be edited will have "(required)"
+within their title.
 
 ### Device Profile
 
-BACnet Device profile can be found at:
-[configs/profiles/bacnet-simulator.json](../config/profiles/bacnet-simulator.json)
+A sample BACnet device profile can be found on the [BACnet/IP XRT Example](../../DeviceServices/bacnet-ip/deployment/profiles/bacnet-ip-sim-profile.json).
+
+An example profile is already included on the [Example desired-bacnet.json](../twin/desired-bacnet.json)
```
[the remark this answers is not in the document]
Comment (amaan-akram) [reply AzureSphere/docs/bacnet-example.md:56]: First I point to an example bacnet profile that is normally used in XRT, this gives the user a clean example of how to create a profile relating it back to core XRT.

Then I mention that the azure examples already have an example profile so there is no need to add a new one if wanting to run this specific example, however for refrence I think it's still good to point to the main bacnet configurations.

Thoughts on this?

[the remark this answers is not in the document]
Comment (BinduRao2018) [reply AzureSphere/docs/bacnet-example.md:56]: Got it. There is no harm in adding additional info. Ignore my comment.

File: AzureSphere/docs/ubuntu-installation.md (hunk 2 above)
Comment (amaan-akram) [reply AzureSphere/docs/ubuntu-installation.md:48]: Not sure, should probably follow the same versioning style as all our other packages

File: AzureSphere/docs/windows-installation.md (hunk 3 above)
Comment (amaan-akram) [reply AzureSphere/docs/windows-installation.md:36]: If the user already had an installation of XRT for Azure and wish to update to a newer release.

added some more points in text for clarification

File: AzureSphere/docs/ethernetip-example.md [hunk 5]
```diff
@@ -1,111 +1,156 @@
...
 
 ### Device Profile
 
-A sample Ethernet-IP device profile for an Allen Bradley PLC can be found at: [configs/profiles/Allen-Bradley-Azure.json](../config/profiles/Allen-Bradley-Azure.json).
+A sample EtherNet/IP device profile can be found on the [EtherNet/IP XRT Example](../../DeviceServices/ethernet-ip/deployment/profiles/ethernetip-sim-profile.json).
+
+An example profile is already included on the [Example desired-ethernetip.json](../twin/desired-ethernetip.json)
 
-The Device profile contains a config of different deviceResources that can be sent or received from the Azure IoT Hub.
+The Device profile contains a config of different deviceResources
```
[the remark this answers is not in the document]
Comment (amaan-akram) [reply AzureSphere/docs/ethernetip-example.md:52]: I will look into cleaning up repeated sections, may need to tidy up and restructure further

File: AzureSphere/docs/ubuntu-installation.md (hunk 2 above)
Comment (BinduRao2018) [reply AzureSphere/docs/ubuntu-installation.md:48]: Could change in 2.1 after discussion. For now, can leave it as is.

File: AzureSphere/docs/ethernetip-example.md [hunk 6]
```diff
@@ -1,111 +1,156 @@
...
+within their title.
 
 ### Device Profile
 
-A sample Ethernet-IP device profile for an Allen Bradley PLC can be found at: [configs/profiles/Allen-Bradley-Azure.json](../config/profiles/Allen-Bradley-Azure.json).
+A sample EtherNet/IP device profile can be found on the [EtherNet/IP XRT Example](../../DeviceServices/ethernet-ip/deployment/profiles/ethernetip-sim-profile.json).
+
+An example profile is already included on the [Example desired-ethernetip.json](../twin/desired-ethernetip.json)
 
-The Device profile contains a config of different deviceResources that can be sent or received from the Azure IoT Hub.
```
Comment (BinduRao2018) [inline AzureSphere/docs/ethernetip-example.md:23]: Info on Device profile should be the first sentence under 'Device Profile'

File: AzureSphere/docs/windows-installation.md [hunk 7]
```diff
@@ -33,5 +33,7 @@ or via the Visual Studio Command Prompt.
 
 ## Updating the XRT package on Windows
 
-When updating the xrt packages with a newer version, it is recomended that you reinstall the azure sphere SDK to ensure we have a clean sysroots directory. After this the same steps can be followed for [installing](#installing-the-xrt-package-on-windows). 
-
+If an outdated version of XRT for azure is already installed and you
```
Comment (BinduRao2018) [inline AzureSphere/docs/windows-installation.md:36]: Above command references azsphere11

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]
2 comments in-line

**Pull request merged**